RELEASE.md

```diff
@@ -168,29 +168,73 @@ point but for now the process is manual.
 1. Open a Powershell terminal in the Windows virtual machine and execute:
 
    ```
-   $ cd c:\users\vagrant\signalfx-agent
-   $ scripts/windows/make.ps1 bundle -AGENT_VERSION "<X.Y.Z>"
+   PS> cd c:\users\vagrant\signalfx-agent
+   PS> .\scripts\windows\make.ps1 bundle -AGENT_VERSION "<X.Y.Z>"
    ```
 
    Where `<X.Y.Z>` is the release version.
 
-1. If the build is successful, verify that
-   `c:\users\vagrant\signalfx-agent\build\SignalFxAgent-X.Y.Z-win64.zip` exists.
+1. If the build is successful, verify that `.\build\SignalFxAgent-X.Y.Z-win64.zip` exists.
 
-1. If everything looks good, run the release script from your local workstation (must be
-   on the Splunk network):
+1. Run the signing script from your local workstation to sign the agent executable in
+   the bundle (must be on the Splunk network):
 
    ```
-   $ scripts/release --stage <STAGE> --push --new-version <X.Y.Z> --component windows --staging-token <repo.splunk.com token> --chaperone-token <chaperone token>
+   $ scripts/signing/sign_win_agent.py --staging-token <repo.splunk.com token> --chaperone-token <chaperone token> build/SignalFxAgent-X.Y.Z-win64.zip
```

Comment:
The instructions regarding `<repo.splunk.com token>` and `<chaperone token>` are unclear. Don't use the term API token. The only place where we refer to "API tokens" is in the SignalFx UI, where we refer to "User API access tokens". These are **session** tokens, so they won't work with the Smart Agent.